### ISSUE TITLE
Poker homework: fix broken specs

### ISSUE THREAD

**Issue by rakvium** [domenic99/trainee-rep#10]
```
trainee-rep/homework$ rspec
F

Failures:

  1) Poker when finder  
     Failure/Error: expect { poker.finder("straight") }.to.contain("Iteration count")

     ArgumentError:
       wrong number of arguments (given 0, expected 1..2)
     # ./spec/poker_spec.rb:7:in `block (3 levels) in <top (required)>'

Finished in 0.00081 seconds (files took 0.2479 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/poker_spec.rb:6 # Poker when finder  
```

**Comment by domenic99** [comment]
Yes, I know that i have wrong specs, also I don't have all tests)

**Comment by rakvium** [comment]
Broken specs fixed in df4e1e3
